packages/jupyter-ai/jupyter_ai/actors/learn.py

```diff
@@ -100,9 +101,11 @@ def delete(self):
             if os.path.isfile(path):
                 os.remove(path)
         self.create()
-    
+
     def create(self):
-        embeddings = OpenAIEmbeddings()
+        embeddings = self.get_embeddings()
+        if not embeddings:
+            return
```

Comment:
This should call `self.reply("Sorry, please select an embedding provider")` or something like that. Don't think we should ever fail silently by default.